test/example/iss/README.md

```diff
@@ -12,9 +12,11 @@ expected outcome.
 The ISS library, `iss.dart`, consists of two classes:
 
 **IssLocator**
+
```

Comment:
Why add these newlines? They make the `<dl>` code (implemented with the `:` on the following line) break.